packages/ts/react-i18n/test/i18n.spec.tsx

```diff
@@ -41,13 +41,29 @@ describe('@vaadin/hilla-react-i18n', () => {
       i18n = new I18n();
       fetchMock
         .get('./?v-r=i18n&langtag=de-DE', {
-          'addresses.form.city.label': 'Stadt',
-          'addresses.form.street.label': 'Strasse',
+          body: {
+            'addresses.form.city.label': 'Stadt',
+            'addresses.form.street.label': 'Strasse',
+          },
+          status: 200,
+          headers: { 'X-Vaadin-Retrieved-Locale': 'de-DE' },
+        })
+        .get('./?v-r=i18n&langtag=de', {
+          body: {
+            'addresses.form.city.label': 'Stadt',
+            'addresses.form.street.label': 'Strasse',
+          },
+          status: 200,
+          headers: { 'X-Vaadin-Retrieved-Locale': 'de-DE' },
```

Comment:
This looks unused?

Reply:
This was a leftover from the older iterations. I removed this case.